### ISSUE TITLE
Exp name containing plus '+' sign does not work

### ISSUE THREAD

**Issue by qizhuli** [torrvision/crayon#26]
My experiment name is 'myexp+suffix'.
When I call the add_scalar_value function for this experiment, I get the following error message:
```
Traceback (most recent call last):
  File "plot.py", line 126, in main
    crayon_experiment.add_scalar_value(key, values[key], step = counter)
  File "anaconda/lib/python2.7/site-packages/pycrayon/crayon.py", line 149, in add_scalar_value
    raise ValueError(msg.format(r.text))
ValueError: Something went wrong. Server sent: Unknown experiment name 'myexp suffix'.
```
It has dropped the + sign from the experiment name and complains experiment does not exist.

**Comment by edran** [comment]
We don't sanitise special characters (such as `+`). Feel free to fork it and write a PR that fixes it if you find the time :)

**Comment by albanD** [comment]
Fixed via #34
